pkg/cmd/server/bootstrappolicy/all.go

```diff
@@ -0,0 +1,106 @@
+package bootstrappolicy
+
+import (
+	"fmt"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/apis/rbac"
+	rbacbootstrappolicy "k8s.io/kubernetes/plugin/pkg/auth/authorizer/rbac/bootstrappolicy"
+
+	authorizationapi "github.com/openshift/origin/pkg/authorization/apis/authorization"
+)
+
+// TODO: this needs some work since we are double converting
```

Comment:
Ok, stupid question (be warned ;)) Why not just express our policies using k8s objects?